config.yaml

```diff
@@ -0,0 +1,13 @@
+# This file defines charm config options, and populates the Configure tab on Charmhub.
+# If your charm does not require configuration options, delete this file entirely.
+#
+# See https://juju.is/docs/config for guidance.
+
+options:
+  # An example config option to customise the log level of the workload
+  broker-rack:
+    description: |
+      Sets the rack id for the broker. 
+
+    default: ""
+    type: string
```

Comment:
**suggestion** can we enforce that this is required, without providing a default? That should also make sure that when deployig this we don't inadvertatively deploy two charms with no-defaults, therefore not resulting in wrong broker awareness

Reply:
The charm can go into blocked status if config is not set.